### ISSUE TITLE
Pan is borked

### ISSUE THREAD

**Issue by Andreya-Autumn** [surge-synthesizer/surge#6623]
Load Init Sine and press latch. 
Move the scene pan slowly towards one extreme. You'll notice the furthest it goes to one side happens around 76-77-ish %, as you keep going towards 100% on the dial it actually comes back around towards the centre.

It gets weirder. Center the pan slider again. Now select the left mixer pan button for the playing oscillator. The sound is heard on the left side. Move the scene pan slider to the left. One would reasonably expect this not to do much, but it brings the sound back to the center. Moving the slider to the right does the same, but makes the combined signal about 6.5 dB quieter.

This is... not good. :P

The pan parameter making any sense would obviously be a breaking change. But Paul proposes a "Legacy Mode" RMB option.

**Comment by mkruselj** [comment]
While at it, don't use the linear taper, use sine or square taper.

**Comment by mkruselj** [comment]
This one seems to be by design, actually...

https://github.com/surge-synthesizer/surge/blob/main/src/common/dsp/vembertech/basic_dsp.h#L312

**Comment by baconpaul** [comment]
Yeah xt2 can use the nice pan laws from mixmaster we have in basic blocks like rack and sc do

**Comment by Andreya-Autumn** [comment]
Makes sense @mkruselj, I figured the stereo widening phase trickery was probably involved. Not such a good choice IMO to make that a function of the pan control, since it gives modulation unpredictable results. 

Proposed XT2 Unbork-my-pan*-plan: 

1. Make the pan param normal. 
2. Relegate any stereo phase tricks to the width slider. 

(*to the tune of unbreak my heart)

**Comment by Andreya-Autumn** [comment]
...and indeed the pan law options from MM seems like a nice option to have. 




Hmm, lines 318 and 327 makes it seem like "normal" pan behavior should live in -75%..75%. But testing it, I'm seeing the deepest pan at around 79.13%... (not quite -∞ there, but I can't be bothered to look for it). So maybe something is a little off there.


If we wanted to unnnnbork my paaaan already in XT1, putting the present behavior behind an "extend range/legacy" kind of dealio might maybe work? I guess that way it could be streamed for older patches to not break. 

And honestly, having that behavior be extended really makes a lot of sense anyway right? It seems very similar to some of the other extendable params, in that most users just wants the normal range most of the time. What do you think?

**Comment by mkruselj** [comment]
I'd much rather just fix it to not do any weird trickery in XT2.
